components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -219,9 +219,15 @@ be lost if SCREAM_HACK_XML is not enabled.
       <nudging_filename type="array(string)"/>
       <nudging_fields type="array(string)"/>
       <nudging_timescale type="integer">0</nudging_timescale>
-      <source_pressure_type type="string">TIME_DEPENDENT</source_pressure_type>
+      <source_pressure_type type="string">TIME_DEPENDENT_3D_PROFILE</source_pressure_type>
```

Comment:
You could add `"valid_values=TIME_DEPENDENT_3D_PROFILE,STATIC_1D_VERTICAL_PROFILE"`, which would catch at submit time if someone has a typo in its atmchange...

Reply:
Also, since we have automatic docs generation, do you mind adding `doc="some descirption"` to the new parameters? :)

Reply:
>You could add "valid_values=TIME_DEPENDENT_3D_PROFILE,STATIC_1D_VERTICAL_PROFILE", which would catch (at submit time) if someone has a typo in its atmchange...

Should I remove the EKAT_ERROR_MSG associated with checking this for a valid value internally then?

Reply:
No, that should stay there, for standalone purposes (as well as for extra protection against xml/inputs tampering). But we want to catch XML errors at buildnml time if possible.